app/talk/popular-tags.cjsx

```diff
@@ -0,0 +1,35 @@
+React = require 'react'
+PromiseRenderer = require '../components/promise-renderer'
+{makeHTTPRequest} = require 'json-api-client'
+{Link} = require 'react-router'
+
+module?.exports = React.createClass
+  displayName: 'TalkPopularTags'
+
+  propTypes:
+    header: React.PropTypes.object             # Optional header ex: <h1>Tags</h1>
+    section: React.PropTypes.string.isRequired # Talk section
+    params: React.PropTypes.object.isRequired # URL params
+
+  tagsRequest: ->
+    popularTags = talkClient.root + "/tags/popular?section=#{@props.section}&limit=10"
+    makeHTTPRequest 'GET', popularTags, null, talkClient.headers
```

Comment:
This was the cleanest I could do. I tried `talkClient.get(...)` first, but since this endpoint returns an array of strings, the response ended up being formatted strangely (I think they were stored in an errant private key )